README.md

```diff
@@ -82,27 +82,36 @@ under GNU General Public License 2 (GPL2).
 ## Table of Contents
 
 - [Specify 7](#specify-7)
-    - [Table of Contents](#table-of-contents)
-    - [Changelog](#changelog)
-    - [Installation](#installation)
-        - [Docker installation](#docker-installation-recommended)
-          (**Recommended**)
-        - [Local installation](#local-installation)
-            - [Installing system dependencies](#installing-system-dependencies)
-            - [Installing Specify 6](#installing-specify-6)
-            - [Cloning Specify 7 source repository](#cloning-specify-7-source-repository)
-            - [Setting up Python Virtual Environment](#setting-up-python-virtual-environment)
-            - [Building](#building)
-            - [Adjusting settings files](#adjusting-settings-files)
-            - [Turning on debugging](#turning-on-debugging)
-            - [The development server](#the-development-server)
-            - [The Specify 7 worker](#the-specify-7-worker)
-            - [Installing production requirements](#installing-production-requirements)
-            - [Setting up Apache](#setting-up-apache)
-            - [Restarting Apache](#restarting-apache)
-    - [Updating Specify 7](#updating-specify-7)
-    - [Updating the database (Specify 6) version](#updating-the-database-specify-6-version)
-    - [Localizing Specify 7](#localizing-specify-7)
+  - [Table of Contents](#table-of-contents)
```

Comment:
nice!
was this done by a vscode plugin?

Reply:
ya, didn't notice that when I pushed the commit.